actions/playerMatchesActions.js

```diff
@@ -0,0 +1,81 @@
+import fetch from 'isomorphic-fetch';
+import { HOST_URL } from './';
+
+const url = (playerId, numMatches) => `/api/players/${playerId}/matches?limit=${numMatches}`;
+
+const REQUEST = 'yasp/playerMatches/REQUEST';
+const OK = 'yasp/playerMatches/OK';
+const ERROR = 'yasp/playerMatches/ERROR';
+
+export const playerMatchesActions = {
+  REQUEST,
+  OK,
+  ERROR,
+};
+
+const getPlayerMatchesRequest = () => ({ type: REQUEST });
+
+const getPlayerMatchesOk = (payload) => ({
+  type: OK,
+  payload,
+});
+
+const getPlayerMatchesError = (payload) => ({
+  type: ERROR,
+  payload,
+});
+
+// TODO - memoize this with selectors
+const playerMatchTransformSwitch = (match, field, heroes) => {
+  switch (field) {
+    case 'hero_id':
+      return `${HOST_URL}${heroes[match[field]].img}`;
+    case 'radiant_win':
+      return match.player_slot < 64 && match[field] || match.player_slot >= 64 && !match[field] ? 'W' : 'L';
```

Comment:
isRadiant is going to be a function used a lot, move it to some utility function file.  Also 128 is more correct